components/script/dom/svgelement.rs

```diff
@@ -13,9 +18,18 @@ use style::element_state::ElementState;
 #[dom_struct]
 pub struct SVGElement {
     element: Element,
+    style_decl: MutNullableDom<CSSStyleDeclaration>,
```

Comment:
also cc @emilio 

I basically refer to `HTMLElement` to add this field. Does this make sense?

Reply:
It does, though maybe it should be moved down to `Element`, even if not web-exposed there, because that's where `style_attribute` lives. Anyhow no big deal.